File: Java/exchange.java
/*
Author: Barbara Anger
2018. Copyright. All Rights reserved.
*/
import java.util.Scanner;

public int exchange(){
    Scanner read = new Scanner(Scanner.read);
    System.out.println("Enter two values:");
    int x = read.nextInt();
    int y = read.nextInt();
    x = x + y;
    y = x - y;
    x = x - y;
    System.out.println("The variables are now:\n");
}

File: C_Language/exchange.c
/*
@author Barbara Anger
2018. Copyright. All Rights reserved.
*/

#include <stdio.h>
int main(){
    int x, y;
    printf("Typ two values:\n");
    scanf("%d", &x);
    scanf("%d", &y);
    x = x + y;
    y = x - y;
    x = x - y;
    printf("The variables are now \n\tx = %d e y = %d\n", x, y);
    return 0;
}